vue.config.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
   publicPath: `/`,
   configureWebpack: () => {
     const config = {
+      devServer: {
```

Comment:
this should go into another PR as it probably breaks things

Reply:
Agree! Was included by error because I use it for testing.